source/css/sass/mixins/_typography.scss

```diff
@@ -4,10 +4,11 @@
 @import "../variables/font";
 @import "spacing";
 @import "utilities";
+@import "scale";
 
 @mixin set-font-size-and-line-height($font-size, $block-size: $baselinegrid-space-small) {
   @include rem(font-size, $font-size);
-  line-height: $block-size / $font-size;
+  line-height: ceil(($block-size / $font-size) * 100000) / 100000;
```

Comment:
This gives better results in _some_ browsers (eg `1.33334` rather than `1.33333` makes a difference).

Reply:
Can't see this anymore? 🤔

Going to revert, can introduce later if needed.